src/partials/contact.html

```diff
@@ -25,7 +25,7 @@
           </div>
         </div>
         <div class="actions">
-          <a class="btn btn-primary btn-lg btn-block" ng-show="userCanEditProfile" href="#/profile/{{contact._id}}" translate>Edit Profile</a>
+          <a class="btn btn-primary btn-lg btn-block" href="#/profile/{{contact._id}}" translate>Edit Profile</a>
```

Comment:
Sorry this was something I was looking into and testing, should not have happened.